de.metas.business.rest-api-impl/src/main/java/de/metas/rest_api/payment/JsonPaymentService.java

```diff
@@ -93,7 +93,7 @@ public JsonPaymentService(final CurrencyService currencyService, final BpartnerP
 			return ResponseEntity.unprocessableEntity().body(String.format("Cannot find Bank Account for bpartner: %s, currency: %s and account: %s", jsonPaymentInfo.getBpartnerIdentifier(), jsonPaymentInfo.getCurrencyCode(), jsonPaymentInfo.getTargetIBAN()));
 		}
 
-		final I_C_Payment payment = paymentBL.newOutboundPaymentBuilder()
+		final I_C_Payment payment = paymentBL.newInboundReceiptBuilder()
```

Comment:
is this always a payment receipt, how can we make sure?

Reply:
We're not sure, but marking it as Inbound was the only way to have it read by allocation in `de.metas.allocation.modelvalidator.C_Invoice.allocateAvailablePayments(I_C_Invoice)`

Reply:
Confirmed with @metas-ts it's ok for now to only create receipts.